group_project_v2/mixins.py

```diff
@@ -104,12 +104,7 @@ def user_id(self):
 
     @lazy
     def user_preferences(self):
```

Comment:
Value of this propery is memoized two times, once with timeout using ``@memoize_with_expiration()`` and once for whole lifespan of the Xblock object (using ``@lazy``), this might not be a very important issue --- but having the same value in two caches should be avoided as they can desynchronize easily.